### PR TITLE
composer kickoff

### PR DESCRIPTION
**Pull request by filipgolonka**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "stp-sa-hackathon/sa-hackathon-backend",
+    "require": {
+        "silex/silex": "~1.2"
+    },
+    "authors": [
+        {
+            "name": "Filip Golonka",
+            "email": "filip.golonka@schibsted.pl"
+        }
```
Comment (rafalkanski) [inline composer.json:10]: and me ?

Comment (marphi) [inline composer.json:10]: and me too!

**Comment by filipgolonka** [comment]
@rafalkanski updated.

**Comment by rafalkanski** [comment]
:+1:

**Pull request merged**